README.md

```diff
@@ -164,6 +164,22 @@ You can put a note on an endpoint using `note`, as in:
 note A: This is a note
 ```
 
+## Blocks
+
+You can create a dotted block boundary around a section using the `block`, `loop`, and `opt`
```

Comment:
I'll generate some examples of these, and perhaps tweak the wording here once I play with them a little.